src/main/java/jeff/CommandHandler.java

```diff
@@ -0,0 +1,123 @@
+package jeff;
+
+import jeff.exceptions.*;
+
+public class CommandHandler {
+    private static final int TODO_DESCRIPTION_INDEX = 5;
+    private static final int DEADLINE_DESCRIPTION_INDEX = 9;
+    private static final int DEADLINE_BY_INDEX_LENGTH = 4;
+    private static final int EVENT_DESCRIPTION_INDEX = 6;
+    private static final int EVENT_FROM_INDEX_LENGTH = 6;
+    private static final int EVENT_TO_INDEX_LENGTH = 4;
+    private static final int MARK_INDEX = 5;
+    private static final int UNMARK_INDEX = 7;
+
+    public static void handleList() {
+        Printer.printTasks();
+    }
+
+    public static void handleTodo(String userInput) throws InvalidTodoSyntaxException {
+        if (userInput.equals("todo")) {
```

Comment:
you can avoid the use of magic literals by storing them as static final vars (constants) or use enums. 

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers